go-controller/pkg/ovn/base_network_controller.go

```diff
@@ -816,7 +816,16 @@ func (bnc *BaseNetworkController) findMigratablePodIPsForSubnets(subnets []*net.
 		}
 		podAnnotation, err := util.UnmarshalPodAnnotation(liveMigratablePod.Annotations, bnc.GetNetworkName())
 		if err != nil {
-			return nil, err
+			// even though it can be normal to not have an annotation now, live
+			// migration is a sensible process that might be used when draining
+			// nodes on upgrades, so log a warning in every case to have the
+			// information available
+			klog.Warningf("Could not get pod annotation of pod %s/%s for NAD %s: %v",
```

Comment:
```suggestion
			klog.Warningf("Could not get pod annotation of pod %s/%s for network %s: %v",
```
?? Or did you actually intend to expose as NAD?

Reply:
Yeah, you are right. 

This is only used for the primary network but is on the base controller for convenience.

Supporting secondary networks here would require iterating though all the NADs of the network, but there might be more gaps. I won't bother with it, it would be useless now. So I will just add a check.

@qinqon PTAL